storage/libradb/src/lib.rs

```diff
@@ -824,6 +855,53 @@ impl DbReader for LibraDB {
         let (version, txn_info) = self.ledger_store.get_latest_transaction_info()?;
         Ok((version, txn_info.state_root_hash()))
     }
+
+    // TODO migrate this to `ConfigStorage` trait as non-async
+    fn batch_fetch_config(&self, access_paths: Vec<AccessPath>) -> Result<Vec<Vec<u8>>> {
```

Comment:
why are we putting this into storage?

Reply:
`ConfigStorage` is a trait to help define on-chain config storage: https://github.com/libra/libra/blob/master/types/src/on_chain_config/mod.rs#L78-L80. I didn't implement `ConfigStorage` for `StorageRead` because I didn't want to pull in async-trait dependency into `types` directory, and was going to migrate this implementation to be a trait implementation once async->sync conversion was complete - was gonna do that after this PR lands